dp/core/dpdk.c

```diff
@@ -59,15 +60,14 @@ int dpdk_init(void)
 	const int pool_size = 32768;
 
 	optind = 0;
-	internal_config.no_hugetlbfs = 1;
+	//internal_config.no_hugetlbfs = 1;
```

Comment:
If line not needed, delete. The Linux kernel coding style says Don't use C99-style "// ..." comments.